### ISSUE TITLE
Unrecognized command: deploy-flexvolume-driver

### ISSUE THREAD

**Issue by marcteyssier** [rancher/longhorn#152]
Hi,

I have an error with the last version of rancher and Longhorn.
So I use the last version of each element:
- Ubuntu 16.04 LTS
- Rancher 2.0.6
- rancher/longhorn-engine: tag 31c42f0
- rancher/longhorn-manager: tag 06a81b9
- rancher/longhorn-ui: tag 47e0b2a

I try many version of longhorn, but it doesn't start :-(

Error log in longhorn-flexvolume-driver-deployer pod

```
panic: Unrecognized command: deploy-flexvolume-driver

goroutine 1 [running]:
main.cmdNotFound(0xc4201a9b80, 0x7ffd3d4f3ad7, 0x18)
        /go/src/github.com/rancher/longhorn-manager/main.go:16 +0xcb
github.com/rancher/longhorn-manager/vendor/github.com/urfave/cli.ShowCommandHelp(0xc4201a9b80, 0x7ffd3d4f3ad7, 0x18, 0x0, 0x0)
        /go/src/github.com/rancher/longhorn-manager/vendor/github.com/urfave/cli/help.go:160 +0x415
github.com/rancher/longhorn-manager/vendor/github.com/urfave/cli.glob..func1(0xc4201a9b80, 0x0, 0x0)
        /go/src/github.com/rancher/longhorn-manager/vendor/github.com/urfave/cli/help.go:89 +0x56
github.com/rancher/longhorn-manager/vendor/github.com/urfave/cli.HandleAction(0x1383ca0, 0x1624158, 0xc4201a9b80, 0x0, 0x0)
        /go/src/github.com/rancher/longhorn-manager/vendor/github.com/urfave/cli/app.go:485 +0xd2
github.com/rancher/longhorn-manager/vendor/github.com/urfave/cli.(*App).Run(0xc420182820, 0xc420010140, 0x5, 0x5, 0x0, 0x0)
        /go/src/github.com/rancher/longhorn-manager/vendor/github.com/urfave/cli/app.go:259 +0x695
main.main()
        /go/src/github.com/rancher/longhorn-manager/main.go:53 +0x44b
```

**Comment by yasker** [comment]
@marcteyssier You cannot just update the yaml file with the new image. Parameters may change.

Please use the yaml file with the specified image.

You can try our newly released v0.3-rc1, following the instruction here:

https://github.com/rancher/longhorn/tree/v0.3-rc

**Comment by yasker** [comment]
Close the issue for now. Please reopen if the solution doesn't work.
